packages/stuntman/build.sh

```diff
@@ -0,0 +1,20 @@
+TERMUX_PKG_HOMEPAGE=http://www.stunprotocol.org/
+TERMUX_PKG_DESCRIPTION="An open source STUN server"
+TERMUX_PKG_LICENSE="Apache-2.0"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=1.2.16
+TERMUX_PKG_SRCURL=http://www.stunprotocol.org/stunserver-${TERMUX_PKG_VERSION}.tgz
+TERMUX_PKG_SHA256=4479e1ae070651dfc4836a998267c7ac2fba4f011abcfdca3b8ccd7736d4fd26
+TERMUX_PKG_DEPENDS="libc++, openssl"
+TERMUX_PKG_BUILD_DEPENDS="boost"
+TERMUX_PKG_BUILD_IN_SRC=true
+TERMUX_PKG_EXTRA_MAKE_ARGS="T="
```

Comment:
Is T= necessary? Seem to work fine without it when I test build

Reply:
May be needed if environment variable `T` is defined. A comment is added. Thanks!